controllers/progressivesync_controller_test.go

```diff
@@ -518,8 +516,8 @@ var _ = Describe("ProgressiveSync Controller", func() {
 
 	Describe("sync application", func() {
 		It("should send a request to sync an application", func() {
-			mockedArgoCDAppClient := mocks.MockArgoCDAppClientCalledWith{}
-			reconciler.ArgoCDAppClient = &mockedArgoCDAppClient
+			mockedArgoCDAppClient := &mocks.MockArgoCDAppClientCalledWith{}
+			reconciler.ArgoCDAppClient = mockedArgoCDAppClient
```

Comment:
```suggestion
			mockedArgoCDAppClient := mocks.MockArgoCDAppClientCalledWith{}
			reconciler.ArgoCDAppClient = &mockedArgoCDAppClient
```
I think I'm missing why we changed this

Reply:
good catch, it's prob a mess with my rebase.